src/easylink/pipeline_graph.py

```diff
@@ -102,6 +83,92 @@ def merge_combined_implementations(self, config):
             except nx.NetworkXNoCycle:
                 pass
 
+    def get_combined_slots_and_edges(
+        self, combined_implementation: str, nodes_to_merge: list[str]
+    ) -> tuple[set[InputSlot], set[OutputSlot], set[EdgeParams]]:
```

Comment:
not sure how consistent we've been, but this seems like a private method?

Should probably have a docstring as well - certainly if it remains public